OJS.Workers.SubmissionProcessors/Workers/RemoteSubmissionsWorker.cs

```diff
@@ -166,6 +166,12 @@ private TResult BuildTestResult<TResult>(TestContext test, TestResultResponseMod
             {
                 result = this.http.PostJson<object, RemoteSubmissionResult>(this.endpoint, submissionRequestBody);
             }
+            catch (ThreadAbortException ex)
+            {
+                submission.ProcessingComment = $"Exception in getting remote submission result: {ex.Message}";
+                submission.ExceptionType = ExceptionType.Thread;
+                throw new Exception($"Exception in {nameof(this.ExecuteSubmissionRemotely)}", ex);
+            }
```

Comment:
I don't see anything different in this catch than in the one below. What's the point of it, besides setting different exception type (which is not needed as mentioned in the previous comment)?